core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -368,6 +368,11 @@ public MetricKey build() {
           .setDescription("Total number of unique blocks in Alluxio")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey MASTER_TOTAL_BLOCK_REPLICA_COUNT =
+      new Builder("Master.TotalBlockReplicaCount")
```

Comment:
I think block replica count should be sufficient
```suggestion
      new Builder("Master.BlockReplicaCount")
```